about.html

```diff
@@ -7,67 +7,96 @@
   <title>Sprint Challenge - About</title>
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="css/index-about.css">
 
```

Comment:
good catch on the link there, i would have maybe done it like you did the contacts css file name and just titled it about.css